### DISCUSSION TITLE
Sharing Tokens with Designers

### DISCUSSION THREAD

**Discussion by mike-ang** [six7/figma-tokens#850] [Q&A]
Hello,

What is the easiest way to share figma tokens with other designers? I'm a designer & noob with github but was able to successfully set up a repository and sync it to my Figma File. I've read about the URL sync in the documentation but not sure how headers or providing read-only access works. How would I go about giving some designers read-only access to the same tokens and others the ability to actually push changes? Thanks in advanced.

**Comment by six7** [comment]
> Hello,
> 
> What is the easiest way to share figma tokens with other designers? I'm a designer & noob with github but was able to successfully set up a repository and sync it to my Figma File. I've read about the URL sync in the documentation but not sure how headers or providing read-only access works. How would I go about giving some designers read-only access to the same tokens and others the ability to actually push changes? Thanks in advanced.

You could invite them to your repository, they could add it as a GitHub provider in the plugin and then they'd be able to push and pull.

If you just want them to read, you can send them the direct link to the file (viewing a file on github on the top right there's a link to `Raw`. They can use that in the URL sync of the plugin.
